src/test/java/uk/gov/hmcts/reform/sscs/refdataloader/AppealStatusReferenceDataLoaderTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.reform.sscs.refdataloader;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import java.io.InputStream;
+import javax.xml.stream.XMLStreamException;
+
+import org.junit.Before;
+import org.junit.Test;
+import uk.gov.hmcts.reform.sscs.models.refdata.AppealStatus;
+
+public class AppealStatusReferenceDataLoaderTest {
+
+    public static final String GAPS2_REFERENCE_DATA_XML =
+        "reference-data/SSCS_Extract_ReferenceTest_2017-06-30-09-01-31.xml";
```

Comment:
Why are you using a different Delta xml file? What's wrong with the existing one? I think we should use the same Delta file to keep things simple.

Reply:
Please read the code carefully. Its for reference data and not delta.

Reply:
sorry, I mean a  different Reference file?

Reply:
The complete reference file is 2.7MB. Not need to use that big file for unit tests